### PR TITLE
change REST to HTTP

### PR DESCRIPTION
**Pull request by bc-pi**
There's nothing necessarily RESTful about this nor should it be limited to REST. Let's just not use the term. 

https://htmx.org/essays/how-did-rest-come-to-mean-the-opposite-of-rest/ is a fun semi-related read, if you have or can find the time.

### CODE REVIEW

**Comment by yaronf** [comment]
Quoting the article:

> I was young at the time, and the whole [discussion] struck me as opaque, puritanical and alienating, so I pretty much gave up on the whole idea of REST: it was something condescending people fought about on the internet.

I am not as young but I agree with this sentiment. Having said that, I guess a standard needs to be reasonably precise in its usage of terminology, and so I accept the PR.

File: draft-sheffer-wimse-s2s-protocol.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ policy management and message authorization are out of scope of this document.
 The high-level message flow is as follows:
 
 * Workload A obtains a credential from the Identity Server. This happens periodically, e.g. once every 24 hours.
-* Workload A makes a REST call into Workload B. This is a regular REST call, with the additional protection
+* Workload A makes an HTTP call into Workload B. This is a regular HTTP call, with the additional protection
```
Comment (yaronf) [inline draft-sheffer-wimse-s2s-protocol.md:117]: I don't think I've ever heard the term "HTTP call". Can we think of a better term?

File: draft-sheffer-wimse-s2s-protocol.md [hunk 2]
```diff
@@ -56,15 +56,15 @@ TODO Abstract
 
 This document defines authentication and authorization in the context of interaction between two workloads.
 This is the core component of the WIMSE architecture {{?I-D.ietf-wimse-arch}}.
-Assume that Service A needs to call Service B. For simplicity, this document focuses on REST services,
+Assume that Service A needs to call Service B. For simplicity, this document focuses on HTTP services,
```
Comment (yaronf) [inline draft-sheffer-wimse-s2s-protocol.md:59]:
```suggestion
Assume that Service A needs to call Service B. For simplicity, this document focuses on HTTP-based services,
```

**Review by yaronf — CHANGES_REQUESTED** [review]

File: draft-sheffer-wimse-s2s-protocol.md (hunk 1 above)
Comment (bc-pi) [reply draft-sheffer-wimse-s2s-protocol.md:117]: I feel like I hear the term used all the time but that doesn't necessarily it's widely used or understood. How about just good ol' "HTTP request"? 

```suggestion
* Workload A makes an HTTP call into Workload B. This is a regular HTTP request, with the additional protection
```

**Pull request merged**